### ISSUE TITLE
New release

### ISSUE THREAD

**Issue by colonelpanic8** [pxqr/udev#4]
Any chance you can release the stuff that has made it on to master but isn't in hackage yet?

**Comment by LeifW** [comment]
This doesn't build on GHC 8.4. Maybe merge https://github.com/pxqr/udev/pull/5 and push a new release?

**Comment by colonelpanic8** [comment]
I'd like to take over this package, since it seems that it is no longer maintained by the owner. I'm going to start going through the process to have this done with hackage. It seems that @pxqr is no longer active on hackage/github, but if you are, please let use know.

**Comment by LeifW** [comment]
I could take it over, too. I have some other code that uses this, and more local changes I could PR. Or we could both maintain it. Whatever you prefer. You have more experience with publishing Haskell packages - I've only published one.

**Comment by colonelpanic8** [comment]
@LeifW Yeah I'm fine with either, just want to get the ball moving on this since I want to have this. I emailed the hackage maintainers already, but if you would rather take it over that would be great.

**Comment by LeifW** [comment]
Done. Published new release to https://hackage.haskell.org/package/udev-0.1.1.0

**Comment by colonelpanic8** [comment]
@LeifW Awesome. Are you going to ressurect the taffybar changes you were doing?
